corehq/apps/app_manager/static/app_manager/js/details/screen.js

```diff
@@ -110,7 +110,7 @@ hqDefine("app_manager/js/details/screen", function () {
         });
         self.showCaseListOptimizations = (
             self.columnKey === 'short' &&
-            hqImport('hqwebapp/js/toggles').toggleEnabled('CASE_LIST_OPTIMIZATIONS') &&
+            initialPageData.get('app_supports_case_list_optimizations') &&
```

Comment:
`app_supports_case_list_optimizations` includes the check for feature flag, so that was removed.